docs/solutions/rhel74_image/settfenv.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/local/bin/python3.6
```

Comment:
I'd be in favor of leaving settfenv.py out for now. It's not necessary to this solution, or specific to it. It might make sense to give users a utility like this, but I believe it should be in a different location.

Reply:
Fine.  Deleted